### PR TITLE
[Insights] Added the Tech preview information

### PR DESCRIPTION
**Pull request by Colin1841**
## Jira Ticket 
https://jira.d2iq.com/browse/D2IQ-87587

## Description of changes being made
Added Technical Preview information.

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4332.s3-website-us-west-2.amazonaws.com/

### CODE REVIEW

File: pages/dkp/kommander/2.2/insights/index.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ enterprise: false
 techPreview: true
 ---
 
-<p class="message--note"><strong>NOTE:</strong> We are offering DKP Insights with this release as a technical preview only.</p>
+<p class="message--note"><strong>NOTE:</strong> We are offering DKP Insights with this release as a [Technical Preview](../legal/version-policy/#experimental-status) only.</p>
```
Comment (Colin1841) [inline pages/dkp/kommander/2.2/insights/index.md:12]:
```suggestion
<p class="message--note"><strong>NOTE:</strong> We are offering DKP Insights with this release as a [Technical Preview](../legal/version-policy/#Technical-preview) only.</p>
```

File: pages/dkp/kommander/2.2/legal/version-policy/index.md [hunk 2]
```diff
@@ -64,6 +64,13 @@ Support for Experimental components is provided on a Standard level, Severity 4
 
 This software is provided "as is" and without any express or implied warranties including, without limitation, the implied warranties of merchantability and fitness for a particular purpose.
 
+### Technical preview
+
+Technical preview, or Tech Preview, is similar to Experimental status, but with the following differences:
```
Comment (mmcelroy91) [inline pages/dkp/kommander/2.2/legal/version-policy/index.md:69]:
```suggestion
Technical Preview, or Tech Preview, is similar to Experimental status, but with the following differences:
```

Comment (mmcelroy91) [reply pages/dkp/kommander/2.2/legal/version-policy/index.md:69]: Not sure if Preview should be capitalized here and below as other references are on the page?  Not title, but other references.

**Review by mmcelroy91 — APPROVED** [review]

**Review by nataliepina — APPROVED** [review]
LGTM other than the comments to address

File: pages/dkp/kommander/2.2/legal/version-policy/index.md [hunk 3]
```diff
@@ -64,6 +64,13 @@ Support for Experimental components is provided on a Standard level, Severity 4
 
 This software is provided "as is" and without any express or implied warranties including, without limitation, the implied warranties of merchantability and fitness for a particular purpose.
 
+### Technical preview
+
+Technical Preview, or Tech Preview, is similar to Experimental status, but with the following differences:
+
+- Tech Previews can be used in Production Environments.
+- The licensing is subject to change in future versions.
```
Comment (cneth) [inline pages/dkp/kommander/2.2/legal/version-policy/index.md:72]: I don't think licensing should be mentioned in the support terms.  They have nothing to do with each other.

File: pages/dkp/kommander/2.2/legal/version-policy/index.md [hunk 4]
```diff
@@ -64,6 +64,13 @@ Support for Experimental components is provided on a Standard level, Severity 4
 
 This software is provided "as is" and without any express or implied warranties including, without limitation, the implied warranties of merchantability and fitness for a particular purpose.
 
+### Technical preview
+
+Technical Preview, or Tech Preview, is similar to Experimental status, but with the following differences:
+
+- Tech Previews can be used in Production Environments.
```
Comment (cneth) [inline pages/dkp/kommander/2.2/legal/version-policy/index.md:71]: I don't thing we want this here.     Keep in mind that the term 'tech preview' may apply to more than just insights over time.   And my understanding is that engineering is not agreeing to provide the current product (insights) with the SLA's necessary to support 'production environments'.   Certainly support is not.

File: pages/dkp/kommander/2.2/legal/version-policy/index.md [hunk 5]
```diff
@@ -64,6 +64,13 @@ Support for Experimental components is provided on a Standard level, Severity 4
 
 This software is provided "as is" and without any express or implied warranties including, without limitation, the implied warranties of merchantability and fitness for a particular purpose.
 
+### Technical preview
+
+Technical Preview, or Tech Preview, is similar to Experimental status, but with the following differences:
```
Comment (cneth) [inline pages/dkp/kommander/2.2/legal/version-policy/index.md:69]: I think it's exactly the same as Experimental, and we should either copy pasta that section here, or add the words technical preview to the 'Experimental' section header.

**Review by cneth — CHANGES_REQUESTED** [review]

File: pages/dkp/kommander/2.2/legal/version-policy/index.md (hunk 4 above)
Comment (Colin1841) [inline pages/dkp/kommander/2.2/legal/version-policy/index.md:71]:
```suggestion
```

File: pages/dkp/kommander/2.2/legal/version-policy/index.md (hunk 5 above)
Comment (Colin1841) [inline pages/dkp/kommander/2.2/legal/version-policy/index.md:69]:
```suggestion
Technical Preview, or Tech Preview, is similar to Experimental status, with the exception that Tech Preview items are considered part of the product and will be developed into future releases.
```

**Comment by kaiwalyajoshi** [comment]
@Colin1841 since the licensing aspects aren't part of Support Policy, I guess there aren't that many differences between them.
We could merge both `Tech Preview` under `Experimental` or just duplicate the two sections with a few changes.

File: pages/dkp/kommander/2.2/legal/version-policy/index.md (hunk 3 above)
Comment (Colin1841) [inline pages/dkp/kommander/2.2/legal/version-policy/index.md:72]:
```suggestion
```

**Review by kaiwalyajoshi — APPROVED** [review]

File: pages/dkp/kommander/2.2/legal/version-policy/index.md [hunk 6]
```diff
@@ -64,6 +64,11 @@ Support for Experimental components is provided on a Standard level, Severity 4
 
 This software is provided "as is" and without any express or implied warranties including, without limitation, the implied warranties of merchantability and fitness for a particular purpose.
 
+### Technical preview
+
+We provide Technical Preview, or Tech Preview, features to showcase capabilities that might be added to future versions of the product. As they are not yet production-ready, the support terms are the same as defined for experimental features. Technical Preview features are not guaranteed to move forward, so could be removed from future versions of the product.```
```
Comment (nataliepina) [inline pages/dkp/kommander/2.2/legal/version-policy/index.md:69]:
```suggestion
We provide Technical Preview, or Tech Preview, features to showcase capabilities that might be added to future versions of the product. As they are not yet production-ready, the support terms are the same as defined for experimental features. Technical Preview features are not guaranteed to move forward, so could be removed from future versions of the product.

Comment (nataliepina) [reply pages/dkp/kommander/2.2/legal/version-policy/index.md:69]: There are some extra backticks here it seems

**Review by cneth — APPROVED** [review]
LGTM as long as you apply Natalie's comment about the superfluous backticks.

**Pull request merged**